README.markdown

```diff
@@ -159,9 +159,9 @@ import BasicJsonProtocol._
 scala> case class Person(name: String, value: Int)
 defined class Person
 
-scala> implicit val personIso = LList.iso(
-         { p: Person => ("name", p.name) :*: ("value", p.value) :*: LNil },
-         { in: String :*: Int :*: LNil => Person(in.head, in.tail.head) })
+scala> implicit val personIso = LList.iso2(
```

Comment:
This is an aside, but maybe iso2 wasn't such a good name.. Would it compile if it's just called `iso`? If not `isoCurried`?